### PR TITLE
Cask Hydrator

### PR DESCRIPTION
**Pull request by awholegunch**
Adds "Cask Hydrator" to the included apps section with two screen-shots, captions, and introduction.

Adds entry to Glossary.

Building at: http://builds.cask.co/browse/CDAP-DDB21-1

### CODE REVIEW

File: cdap-docs/included-applications/source/index.rst [hunk 1]
```diff
@@ -27,12 +27,40 @@ extensible: users can write their own artifacts if they so chose, and can
...
 ETL is **Extract**, **Transform** and **Load** of data, a common first-step in any data
 application. CDAP endeavors to make performing ETL possible out-of-box without writing
 code; instead, you just configure CDAP appropriately and operate it.
 
+In this CDAP release, we have added support for self-service batch and real-time data
+ingestion combined with ETL for the building of Hadoop Data Lakes. Called **Cask Hydrator**, it
+provides CDAP users a seamless and easy way to configure and operate ingestion pipelines
+from different types of sources and data. 
+
+*Cask Hydrator*, part of the :ref:`CDAP-UI <cdap-ui>`, provides an easy method of configuring
```
Comment (sreevatsanraman) [inline cdap-docs/included-applications/source/index.rst]: Can we remove "part of the :ref: CDAP-UI"

File: cdap-docs/reference-manual/source/glossary.rst [hunk 2]
```diff
@@ -194,3 +194,9 @@ Glossary
       formats consist of a :ref:`schema <stream-exploration-stream-schema>` and a :ref:`format
       <stream-exploration-stream-format>` (such as CSV, TSV, or Avro, amongst others).
       
+   Cask Hydrator
+      Cask Hydrator, part of the :term:`CDAP-UI`, provides an easy method of configuring
```
Comment (sreevatsanraman) [inline cdap-docs/reference-manual/source/glossary.rst]: remove "part of the CDAP-UI"

**Comment by sreevatsanraman** [comment]
Fix comment and merge please

File: cdap-docs/included-applications/source/index.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/included-applications/source/index.rst]: Fixed.

File: cdap-docs/reference-manual/source/glossary.rst (hunk 2 above)
Comment (awholegunch) [inline cdap-docs/reference-manual/source/glossary.rst]: Fixed.

**Pull request merged**